### ISSUE TITLE
AssemblyScript Token Contract example

### ISSUE THREAD

**Issue by mikedotexe** [near/devx#55]
Working on getting the CI, GitPod, and local development working for this example from:
https://github.com/nearprotocol/awesome-near

**Comment by mikedotexe** [comment]
Remember to comb for this:
Use WalletConnection :
  window.walletConnection = new nearlib.WalletConnection(window.near, nearConfig.contractName);
Use walletConnection.account() to have Account instance which can request signing from wallet:
  contract = new nearlib.Contract(walletConnection.account(), nearConfig.contractName, {
    viewMethods: ['getMessages'],
    changeMethods: ['addMessage'],
    sender: window.accountId,
  });
basically stuff from here https://studio.nearprotocol.com/?f=mdqekeqkd

**Comment by mikedotexe** [comment]
> Remember to comb for this:
> Use WalletConnection :
> window.walletConnection = new nearlib.WalletConnection(window.near, nearConfig.contractName);
> Use walletConnection.account() to have Account instance which can request signing from wallet:
> contract = new nearlib.Contract(walletConnection.account(), nearConfig.contractName, {
> viewMethods: ['getMessages'],
> changeMethods: ['addMessage'],
> sender: window.accountId,
> });
> basically stuff from here https://studio.nearprotocol.com/?f=mdqekeqkd

moved into it's own ticket
